compiler/testSuite/14-array/fixedlength1-p.bal

```diff
@@ -0,0 +1,4 @@
+public function main() {
+    int[4] v = [1, 2];
+    v[3] = 4; // @panic no filler value
```

Comment:
Should add filler values

Reply:
Can you please open a separate issue for this?

Reply:
Added in #946